### PR TITLE
Redirecting vault page on first load to invest page

### PR DESCRIPTION
**Pull request by 0xSero**
Issue:

When refreshing window or starting app on Vault page user is unable to access their information. Even if the users' metamask is connected to the website / the user clicks the connect button.

Solution:

In MainRouter I added a useEffect hook to check the users' current location, if the user loaded the page on Vault then we will redirect the user to invest page and refresh the page.

This all happens within 0.5s so it shouldn't be a problem for performance or data usage.

### CODE REVIEW

File: src/containers/MainRouter/index.tsx [hunk 1]
```diff
@@ -16,6 +16,14 @@ const SentryRoute = Sentry.withSentryRouting(Route);
 
 const MainRouter: FC<IProps> = () => {
   const location = useLocation();
+  const history = useHistory();
+  useEffect(() => {
+    if (location.pathname === '/vaults') {
+      history.push('/');
```
Comment (abhisheknaiidu) [inline src/containers/MainRouter/index.tsx:22]: instead of `history.push('/')` , let's add Routes from `constants` file => `history.push(Routes.Invest)`

**Review by abhisheknaiidu — CHANGES_REQUESTED** [review]

**Review by abhisheknaiidu — APPROVED** [review]
works perfectly now! ✨

**Comment by Seroxdesign** [comment]
@mikeghen Can you please review this, it stops the app from breaking if user tries to load from Vault page (Currently an issue in app.ricochet)

**Comment by mikeghen** [comment]
Hm, is this the right thing to do though? Should we really redirect someone when they refresh on the Vault page? I have seen this issue but I would think the right thing to do would be to fix it so when they're on the vault page it either loads their vaults or it says you don't have any vaults yet

**Comment by Seroxdesign** [comment]
No problem, will re-work this issue then.

**Pull request closed without merging**